### PR TITLE
all: Delete some redundant code

### PR DESCRIPTION
**Pull request by bogem**
Code looks a little bit cleaner now

### CODE REVIEW

**Comment by bogem** [comment]
Any questions?

**Comment by bep** [comment]
How is this an improvement?

**Comment by bogem** [comment]
Haha. This is really not an improvement (except of simplifying generating of hugo version ([code](https://github.com/spf13/hugo/pull/2628/files#diff-3793dfaf7c918125b94a1f1d0baf59d4L39))). But the code is a liiiiiiittle bit cleaner now 😅

**Comment by bep** [comment]
>But the code is a liiiiiiittle bit cleaner now

How is that? I assume the return values from most of these are of type `error` -- and while I agree that not handling these error is bad and should fixed, just removing the "_" is more like sweeping the bugs under the carpet.

**Comment by bogem** [comment]
Ok. Then I will edit it and will check error instead of just ignoring it

File: commands/genman.go [hunk 1]
```diff
@@ -36,7 +35,7 @@ in the "man" directory under the current directory.`,
 		header := &doc.GenManHeader{
 			Section: "1",
 			Manual:  "Hugo Manual",
-			Source:  fmt.Sprintf("Hugo %s", helpers.HugoVersion()),
```
Comment (bep) [inline commands/genman.go]: I think the fmt version is easier to understand.

Comment (bogem) [inline commands/genman.go]: But I think the concatenation of two simple strings is also not so hard to understand.

Comment (bep) [inline commands/genman.go]: So both are easy, why change it?

Comment (bogem) [inline commands/genman.go]: Because it's a bit faster😅

**Comment by bogem** [comment]
Ok, this PR was not good idea. I promise, this PR was the last mediocre contribution

**Comment by bep** [comment]
@bogem the potential error fixes is a good idea, but I don't like cosmetic changes just for changes sake, typos are easy to make, and this is how bugs happen.

**Pull request closed without merging**